### ISSUE TITLE
Add Travis CI config

### ISSUE THREAD

**Issue by mbrukman** [JanusGraph/janusgraph-foundationdb#19]
I have [enabled this repo on Travis CI](https://travis-ci.org/github/JanusGraph/janusgraph-foundationdb).

What we need now is a PR with a small `.travis.yml` file that will build and test the code, before we start making any cleanup or other changes here.

You can see what is currently done for the [main JanusGraph repo](https://github.com/JanusGraph/janusgraph/blob/master/.travis.yml), but we don't need anything close to that level of complexity, we can have something much smaller and simpler to start with here.

**Comment by farodin91** [comment]
I think would better to start switching to the travis-ci.com.

**Comment by mbrukman** [comment]
FWIW, I went to https://travis-ci.com/github/JanusGraph/janusgraph-foundationdb first, but even after signing in, it says:

> We couldn't display the repository
> JanusGraph/janusgraph-foundationdb
>
> This repository may not exist or you may need to use another account to have access to it
>
> Sign in with another account

which is why I then went to https://travis-ci.org/github/JanusGraph/janusgraph-foundationdb and activated it there.

---

It looks like we have to do a migration from travis-ci.org to travis-ci.com; if I visit https://travis-ci.com/organizations/JanusGraph/repositories, I see the following:

![image](https://user-images.githubusercontent.com/4074659/87673443-946a8800-c742-11ea-87cd-9d68b6597630.png)

---

While it might be a good idea to start using travis-ci.com, I think it's a much bigger change, and we will end up migrating the entire org at the same time, and we will also switch from using the webhook to using Travis as a GitHub App, and that's bound to introduce new changes and potential complications.

I would propose we decouple the addition of a `.travis.yml` to this repo (which is blocking cleanup and other enhancement work) from migrating to travis-ci.com and also the migration to the Travis-as-GitHub-App. We should probably do the test with the main repo first (github.com/JanusGraph/janusgraph) to ensure that it works there, and if it does, it should work for all other smaller, less complex repos as well.

Thoughts?
